website/docs/r/instance_pool.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "exoscale"
+page_title: "Exoscale: exoscale_instance_pool"
+sidebar_current: "docs-exoscale-instance-pool"
+description: |-
+  Provides an Exoscale instance pool resource.
+---
+
+# exoscale\_instance\_pool
+
+Provides an Exoscale `instance pool` resource. This can be used to create, modify, and delete instance pools.
+
+
+## Example Usage
+
+```hcl
+resource "exoscale_ssh_keypair" "key" {
+  name = "terraform-mywebapp-keypair"
+}
+
+variable "zone" {
+  default = "de-fra-1"
+}
+
+data "exoscale_compute_template" "mywebapp" {
+  zone = "${var.zone}"
+  name = "mywebapp"
+  filter = "mine"
+}
+
+resource "exoscale_instance_pool" "webapp" {
+  zone = "${var.zone}"
+  name = "webapp"
+  template_id = "${data.exoscale_compute_template.mywebbapp.id}"
+  size = 3
+  service_offering = "Medium"
+  disk_size = 50
+  description = "This is the production environment for my web app"
+  user_data = "#cloud-config\npackage_upgrade: true\n"
+  key_pair = "${exoscale_ssh_keypair.key.name}"
+
+  security_group_ids = ["4d388ced-209d-4be4-932c-f99d14d6e3b9"]
+  network_ids = ["13ec3ed2-ec06-4061-9bd3-92bd0e3adebf"]
+
+  timeouts {
+    create = "10m"
+  }
+}
+```
+
+## Argument Reference
+
+* `zone` - (Required) The name of the [zone][zone] to deploy the Compute instance into.
+* `name` - (Required) The name of the instance pool.
+* `template_id` - (Required) The ID of the Compute instance [template][template]. Usage of the [`compute_template`][compute_template] data source is recommended.
+* `size` - (Required) The number of instances in the instance pool.
+* `service_offering` - (Required) The Compute instance [size][size], e.g. `Tiny`, `Small`, `Medium`, `Large` etc.
+* `disk_size` - The instances disk size from the instance pool.
+* `description` - The description of the instance pool.
+* `user_data` - A [cloud-init][cloudinit] configuration. Whenever possible don't base64-encode neither gzip it yourself, as this will be automatically taken care of on your behalf by the provider.
```

Comment:
```suggestion
* `user_data` - A [cloud-init][cloudinit] configuration to apply when creating Compute instances. Whenever possible don't base64-encode neither gzip it yourself, as this will be automatically taken care of on your behalf by the provider.
```